### PR TITLE
[node-telegram-bot-api] Updated new methods to release v0.50.0

### PR DESCRIPTION
**Pull request by dancingpizza**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:

> - [sendDice](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+sendDice
> )
> - [setChatPermissions](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+setChatPermissions)
> - [getMyCommands](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#telegrambotgetmycommandsoptions--promise)
> - [setMyCommands](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#telegrambotsetmycommandscommands-options--promise)
> - [setChatAdministratorCustomTitle](https://github.com/yagop/node-telegram-bot-api/blob/master/doc/api.md#TelegramBot+setChatAdministratorCustomTitle)

- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/node-telegram-bot-api/index.d.ts [hunk 1]
```diff
@@ -1388,6 +1405,26 @@ declare class TelegramBot extends EventEmitter {
...
+    options?: TelegramBot.SendDiceOptions
+  ): Promise<TelegramBot.Message>;
+
+  setChatAdministratorCustomTitle(
+    chatId: number | string,
+    userId: string,
+    customTitle: string
+  ): Promise<boolean>;
+
+  getMyCommands(): Promise<TelegramBot.BotCommand>;
```
Comment (XC-Zhang) [inline types/node-telegram-bot-api/index.d.ts:1425]: Is it `TelegramBot.BotCommand` or `Array<TelegramBot.BotCommand>`? The [docs](https://core.telegram.org/bots/api#getmycommands) says:

> Returns Array of BotCommand on success.

**Review by XC-Zhang — CHANGES_REQUESTED** [review]
Thanks for the PR! Please see those pending comments. Besides comments, it looks like you use 2 spaces for indentation but the original code uses 4 spaces. For consistency, please use 4 spaces. 😄

File: types/node-telegram-bot-api/index.d.ts [hunk 2]
```diff
@@ -286,6 +287,17 @@ declare namespace TelegramBot {
         error_message?: string;
     }
 
+  interface SendDiceOptions {
```
[the remark this answers is not in the document]
Comment (ariutin) [reply types/node-telegram-bot-api/index.d.ts:290]: > Besides comments, it looks like you use 2 spaces for indentation but the original code uses 4 spaces. For consistency, please use 4 spaces. 😄
Prettier good as always 🙈

File: types/node-telegram-bot-api/index.d.ts (hunk 1 above)
Comment (ariutin) [reply types/node-telegram-bot-api/index.d.ts:1425]: Yeah, my bad, i'll fix it.

File: types/node-telegram-bot-api/index.d.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ariutin) [reply types/node-telegram-bot-api/index.d.ts]: > Will it be better to extend `SendBasicOptions`

SendBasicOptions

**Comment by ariutin** [comment]
@XC-Zhang I had fixed fixed the issues 😇

**Review by XC-Zhang — APPROVED** [review]
Greate job! Thank you for contribution! 😆

**Comment by ariutin** [comment]
Ready to merge

**Pull request merged**